emuu-client/src/components/upload/UploadButton.js

```diff
@@ -102,22 +95,20 @@ function FileUpload() {
       alert("Please upload a video first");
     }
     //Restrict file size to 40 MB ~ equivalent to 30 second video
-    if (file.size > 40 * 1024 * 1024) {
+    if (file.size > 5 * 1024 * 1024) {
```

Comment:
This file size is too small. A 1080p 30 second video is roughly 40mb. To use 5 mb would be limited to 240p to 480p